apps/www/src/lib/registry/default/ui/form/index.ts

```diff
@@ -4,3 +4,4 @@ export { default as FormLabel } from './FormLabel.vue'
 export { default as FormControl } from './FormControl.vue'
 export { default as FormMessage } from './FormMessage.vue'
 export { default as FormDescription } from './FormDescription.vue'
+export * from './injectionKeys'
```

Comment:
Let's do this instead please

```suggestion
export { FORM_ITEM_INJECTION_KEY } from './injectionKeys'
```